nozzle.py

```diff
@@ -320,114 +308,131 @@ def main(ctx_factory=cl.create_some_context, casename="nozzle", user_input_file=
     #   298 K
     #   rho = 1.77619667e-3 kg/m^3
     #   velocity = 0,0,0
-    rho_bkrnd=1.77619667e-3
-    pres_bkrnd=100
-    temp_bkrnd=298
-     
+    rho_bkrnd = 1.77619667e-3
+    pres_bkrnd = 100
+    temp_bkrnd = 298
+
     # nozzle inflow #
-    # 
+    #
     # stagnation tempertuare 298 K
     # stagnation pressure 1.5e Pa
-    # 
-    # isentropic expansion based on the area ratios between the inlet (r=13e-3m) and the throat (r=6.3e-3)
+    #
+    # isentropic expansion based on the area ratios between the inlet (r=13e-3m)
+    # and the throat (r=6.3e-3)
     #
     # calculate the inlet Mach number from the area ratio
-    nozzleInletRadius = 13.e-3
+    nozzleInletRadius = 13.0e-3
```

Comment:
Camel case? 😱

Reply:
I struggled with this too when first getting into CEESD (python general) style. 

My inkling was to replace all with underscores and lower case, but then the names are too long - easily running afoul of our preferred line-length limit, and very awkward looking expressions.

What I usually end up with is the minimally descriptive variable names that I hate to see when reading code, but have learned to live with it.  After accepting that (np == numpy), saving 3 letters but making it impossible to tell at a glance what `np` really is, I've lost my right to complain about descriptive naming.

`inlet_rad` works here?

Reply:
I removed the camel case (boo!)